code/index.html

```diff
@@ -2,21 +2,246 @@
 <html>
   <head>
     <meta charset="utf-8" />
-    <title>Project 2: The News Site</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Save the planet</title>
     <link rel="stylesheet" href="./style.css" />
     <!-- dont forget the meta tag for responsiveness -->
   </head>
   <body>
     <header>
       <!-- Some kind of logo and navbar -->
+      <div class="navigation"> 
+        <div class="hamburger">
+          <div></div>
+          <div></div>
+          <div></div>
+        </div>
+        <img class="logo" src="Penguin_logo.svg" alt="logo">
+        <nav class="right-nav">
+          <a href="About">About</a>  
+          <a href="Contact">Contact</a>  
+          <a href="Get involved">Sign up</a>
+        </nav>
+      </div>
     </header>
 
-    <section class="big-news">
-      <!-- The big news content -->
-    </section>
 
-    <section class="">
-      <!-- The other news  -->
-    </section>
-  </body>
+<!-- The big news content -->
+  <section class="big-news">
+    <img src="./lion.png" alt="">
+    <div class=head-line>
+      <div class="slogan">
+        <h1>SAVE YOUR PLANET TODAY!</h1>
+        <p>Yes. It is still possible</p>
+      </div>
+      <div class="buttons">
+        <button>Read more</button>
+      </div>
+    </div>
+  </section>
+
+<!-- SLOGAN --->
+  <section class="other-news">
+    <div class="large-box">
+      <div class="small-box">
+      <img src="./bird.png" alt="">
+      </div>
+    </div>
+    <div class="other-news-text">
+      <h2>We're all a part of this amazing planet, but we're changing it like never before. Discover the story of the one place we all call home.</h2>
+    </div>
+  </section>
+
+<!-- SMALL NEWS  -->
+
+    <div class="small-news-heading">
+      <h2>Our latest news</h2>
+    </div>
+
+  <section class="small-news">
+    <div class="box-one">
+        <div class="photo">
+          <img src="./antartica.png" alt="antartica">
+          </div>
+        <div class="minihead-line">
+            <a href="https://www.nationalgeographic.com/magazine/2018/11/antarctica-climate-change-western-peninsula-ice-melt-krill-penguin-leopard-seal/">
+              <div class="text">
+              <h3>The big meltdown</h3>
+              <p>Scientist are alarmed by the rapid changes</p>  
+            </a>
+          </div>
+        </div>
+    </div>
```

Comment:
Hi Sara! Still some indentation changes that need to be done. Here is an example:

```
<div class="box-one">
  <div class="photo">
    <img src="./antartica.png" alt="antartica">
  </div>
  <div class="minihead-line">
    <a href="https://www.nationalgeographic.com/magazine/2018/11/antarctica-climate-change-western-peninsula-ice-melt-krill-penguin-leopard-seal/">
      <div class="text">
        <h3>The big meltdown</h3>
        <p>Scientist are alarmed by the rapid changes</p>  
      </div>
    </a>
  </div>
</div>
```

Reply:
Omg Can't believe I am so bad at indentations. I tried now look through each line and used format HTML language.  For me now it looks good, but maybe I missed something🙈